asapdiscovery-docking/asapdiscovery/docking/tests/test_docking_v2.py

```diff
@@ -31,7 +47,7 @@ def test_docking_dask(docking_input_pair, docking_input_pair_simple, use_dask):
     os.getenv("RUNNER_OS") == "macOS", reason="Docking tests slow on GHA on macOS"
 )
 def test_docking_with_file_write(docking_input_pair_simple, tmp_path):
-    docker = POSITDocker()
+    docker = POSITDocker(use_omega=False)  # save compute
     results = docker.dock([docking_input_pair_simple])
     assert results[0].probability > 0.0
     docker.write_docking_files(results, tmp_path)
```

Comment:
this isn't exactly the point of this PR, but while you're improving these tests I wonder if we can save the output of `test_docking` and just have a separate test for just `docker.write_docking_files(results, tmp_path)`

Reply:
Yeah a good point, I'll make change.

Reply:
Doneskies.